GiniVision/Classes/Camera.swift

```diff
@@ -66,10 +66,11 @@ internal class Camera {
                 return completion({ _ in throw CameraError.noInputDevice })
             }
             // Set the orientation accoding to the current orientation of the device
+            
             if let orientation = AVCaptureVideoOrientation(self.motionManager.currentOrientation) {
                 connection.videoOrientation = orientation
             } else {
-                connection.videoOrientation = .portrait
+                connection.videoOrientation = AVCaptureVideoOrientation(UIApplication.shared.statusBarOrientation)
```

Comment:
having a dependency on `UIApplication` is not ideal, IMHO. Also, this is the status bar orientation, not the device orientation. I don't remember correctly, but I think there might be a case when this is not 100% what you want.